docs/userEquitiesDescription.json

```diff
@@ -0,0 +1,67 @@
+index -> userLogin
+type -> user
+
+{
+	"email": "user@example.com",
+	"password": "XXXXX", //To be hashed
+	
+	"tokens": [
+		"token1",
+		"token2"
+	]
+
+}
+
+index -> userEquities
+type -> userEquity
+
+{
+	"user": "user@example.com",
+	"equity": "IAG",
+
+	"actualEquities": 150,
+	"averageEquityCost": 5.71,
+
```

Comment:
Stock options would be considered at purchases but with zero cost? (as they're a different equity)

Reply:
Well, we can consider script dividends, purchases with zero cost. The bad part is that we will lose the concept of "dividend" and income by dividend.

May be I will split the dividends into script ad non-script. I'll think about that.

Reply:
Take into account that the equity will be (i.e.) "OP.INGES-FEB" instead of "INGES" or whatever, but yeah, we would definitively need a flag or similar to discern both.